src/components/dictionaryConcepts/components/ConceptNameRows.jsx

```diff
@@ -129,6 +129,7 @@ class ConceptNameRows extends Component {
         </td>
         <th scope="row" className="concept-language">
           <Select
+            id="locale_full"
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, am supplying the button with a unique identifier in order to test it.